packages/vulcan-core/lib/modules/containers/withMulti.js

```diff
@@ -63,11 +63,17 @@ export default function withMulti(options) {
   //pollInterval = typeof window === 'undefined' ? null : pollInterval;
 
   const { collectionName, collection } = extractCollectionInfo(options);
-  const { fragmentName, fragment, extraVariablesString } = extractFragmentInfo(options, collectionName);
+  const { fragmentName, fragment } = extractFragmentInfo(options, collectionName);
 
   const typeName = collection.options.typeName;
   const resolverName = collection.options.multiResolverName;
 
+  // LESSWRONG MODIFICATION: Allow the passing of extraVariables so that you can have field-specific queries
```

Comment:
Hmm, you seem to make this modification only in withMulti, which means it likely doesn't work in `withSingle`, `withDelete`, `withCreate`, etc.?

Reply:
It already existed in `withSingle`; this is more-or-less a copy-paste of that

Reply:
Ah, cool. Then that seems good.